Makefile

```
@@ -27,6 +27,10 @@ appengine-tests:
 vulnfeed-tests:
 	cd vulnfeeds && ./run_tests.sh
 
+api-server-tests:
+	test $(HOME)/.config/gcloud/application_default_credentials.json || (echo "GCP Application Default Credentials not set, try 'gcloud auth login --update-adc'"; exit 1)
+	cd gcp/api && ./run_tests.sh $(HOME)/.config/gcloud/application_default_credentials.json
```

```suggestion
	test -f ${HOME}/.config/gcloud/application_default_credentials.json || (echo "GCP Application Default Credentials not set, try 'gcloud auth login --update-adc'"; exit 1)
	cd gcp/api && ./run_tests.sh ${HOME}/.config/gcloud/application_default_credentials.json
```

(I don't actually know bash too well, I was just trying to manually run the commands on my machine)

---

Good call that is more specific